linkerd/dns/Cargo.toml

```diff
@@ -16,6 +16,6 @@ tokio = { version = "1", features = ["rt", "sync", "time"] }
 pin-project = "1"
 
 [dependencies.trust-dns-resolver]
-version = "0.20.0"
+version = "=0.20.1"
```

Comment:
Would be nice to be able to remove this...maybe worth a comment about what upstream changes we are waiting on before we can remove the `=`?